src/index.js

```diff
@@ -191,18 +191,28 @@ if (!fmt) {
 // main loop
 let exitValue = 0;
 const results = rootProject.findIssues(options.opt.locales);
+let errors = 0;
+let warnings = 0;
 
 results.forEach(result => {
     const str = fmt.format(result);
     if (str) {
         if (result.severity === "error") {
             logger.error(str);
             exitValue = 2;
-        } else if (!options.opt.errorsOnly) {
-            logger.warn(str);
-            exitValue = Math.max(exitValue, 1);
+            errors++;
+        } else {
+            warnings++;
+            if (!options.opt.errorsOnly) {
+                logger.warn(str);
+                exitValue = Math.max(exitValue, 1);
+            }
         }
     }
 });
 
+if (results.length) {
+    logger.info(`Errors: ${errors}, Warnings: ${warnings}`);
```

Comment:
Even if a user set `errosOnly`, does the warning count shows as well?

Reply:
Yes, it will show the warning count, but not print the warnings on the screen. Do you think we should only print the error count if the errorsOnly flag is given?

Reply:
IMHO yes. wouldn't it be better to not show any warning stuff message when errorOnly flag is true?